packages/llama/src/chain-ids.ts

```diff
@@ -0,0 +1,3 @@
+export const OPTIMISM_CHAIN_ID = 10
+
+export const CHAIN_ID_ARRAY = [OPTIMISM_CHAIN_ID]
```

Comment:
Llama will be on both Arbitrum and Optimism. We should add support for both.

Reply:
I ended up adding every chain they support expect testnets!